packages/devtools_app/lib/src/memory/memory_allocation_table_data.dart

```diff
@@ -123,23 +127,25 @@ class FieldSizeColumn extends ColumnData<ClassHeapDetailStats> {
   }
 
   @override
-  double get fixedWidthPx => fieldWidthNumbers;
+  double get fixedWidthPx => defaultNumberFieldWidth;
 }
 
-class FieldSizeAccumulatorColumn extends ColumnData<ClassHeapDetailStats> {
-  FieldSizeAccumulatorColumn()
+class FieldSizeDeltaColumn extends ColumnData<ClassHeapDetailStats> {
+  FieldSizeDeltaColumn()
       : super(
-          'Accum',
+          'Delta',
           alignment: ColumnAlignment.right,
         );
 
   @override
-  dynamic getValue(ClassHeapDetailStats dataObject) =>
-      dataObject.bytesAccumulated;
+  int getValue(ClassHeapDetailStats dataObject) =>
+      dataObject.bytesDelta;
 
+  // TODO(terry): Only show grows (negative value returns 0). Consider setting
+  //              to display growth and decline.
   @override
   String getDisplayValue(ClassHeapDetailStats dataObject) =>
-      '${getValue(dataObject)}';
+      '${max(0, getValue(dataObject))}';
```

Comment:
why is there a max? I would have thought a negative delta is fine?

Reply:
Here's the comment I left to Youyang about Monitor/Reset.

Reset is a technique that the VM/Observatory used, which our user study folks liked. Basically, clicking 'Reset' will reset accumulators to zero the next time a "Monitor Allocation" is clicked the Accumulators will show the number of objects allocated between last "Reset" and "Monitor Allocation". We could also display "Accumulators" as "Diff". I currently abbreviate Accumulators as "Accum" could also use "ACC". But I don't have a lot of horizontal space to have a column heading of "Accumulators". Any suggestions on this problem? Jacob suggestion "Delta" or "Difference". I like Delta (it's short) I'll use that.

It only show growth (positive values negatives values displays as zero this is what observatory did) I think that's more important, to show leaks. Maybe we could have a setting that defaults to only show growth unless turned off then displays both positive and negative growth?

Reply:
If we show negative values they should definitely be sorted at the end.
I think there is some prior art for showing negative deltas as well. For example I believe this tool supports it.
https://www.yourkit.com/docs/java/help/compare_snapshots.jsp
Negative deltas aren't typically as actionable but could be useful if you want to sanity check that a bunch of memory was freed like you expected when for example navigating to a different route in your flutter app.

Reply:
I agree I'll add a TODO.  What do we think about adding this is a setting?